src/components/table.js

```diff
@@ -1,43 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Table as AntdTable } from 'antd';
-import { useTranslation } from 'react-i18next';
-
-function Table({ className, pagination, ...tableProps }) {
-  const { t } = useTranslation('table');
-
-  const itemRender = (current, type, originalElement) => {
-    switch (type) {
-      case 'prev':
-        return <span className="Table-paginationItemContent Table-paginationItemContent--prevNext">{t('prev')}</span>;
-      case 'next':
-        return <span className="Table-paginationItemContent Table-paginationItemContent--prevNext">{t('next')}</span>;
-      case 'page':
-        return <span className="Table-paginationItemContent Table-paginationItemContent--page">{current}</span>;
-      default:
-        return originalElement;
-    }
-  };
-
-  return (
-    <AntdTable
-      className={['Table', className].filter(x => x).join(' ')}
-      rowClassName={() => 'Table-row'}
-      Pagination={!!pagination && { position: ['bottomCenter'], showSizeChanger: false, itemRender }}
```

Comment:
This no longer had any effect since the AntD version update. We cannot affect the table pagination from the outside anymore, except for a single option, `position`.

To streamline table usages, I've turned this into a simple `DocumentsTable` component (having a custom component also called `Table` just like the AntD one made it confusing). I've also dropped the customizations of the paging, because with it we were now straying too much from AntD styling and we should probably keep it simple and use the styles that it has. This also keeps the website look and feel more consistent "ant" style, I would say.

Reply:
Ha! I actually completely forgot that we have our own `Table` (only stumbled upon it yesterday). Yeah, let's use the provided pagination and not meddle too much in it. Given the screenshots it is not really worth the difference.